libs/search-widget/src/viewer/paragraphs/Paragraphs.svelte

```diff
@@ -17,6 +19,10 @@
       paragraph: paragraph.paragraph,
     });
   };
+  const showLabels = nucliaState().widget.pipe(
```

Comment:
I think we will always show labels (no matter if edition is enabled or not)

Reply:
ok, now they are always shown